spec/restaurant_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'restaurant'
+
+describe Restaurant do
+  let(:menu_item){double(:menu_item, name: 'chicken satay', price: 4.50)}
+  let(:order){double(:order, :add_item => true)}
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)